src/pe-three-tier-esc-header/widget/pe-three-tier-esc-header.u-css.scss

```diff
@@ -0,0 +1,171 @@
+.navbar {
+    background-color: $navbar-inverse-bg;
+    border-radius: 0;
+    border: 0;
+    margin-bottom: 0;
+  }
+  
+  .navbar-row {
+    max-width: 1276px;
+    margin: 0 auto;
+  }
+  
+  .fa-home {
+    font-size: 25px;
+    color: $navbar-inverse-link-color;
+    margin-left: 30px;
+  }
+  
+  .navbar-subnav {
+    width: 100%;
+    height: auto;
+    border-bottom: 1px solid $light-gray;
+    background-color: $tier-2-background;
+  }
+  
+  .subnav-container {
+    height: 175px;
+  }
+  
+  .center-logo {
+    text-align: center;
+    margin: 40px 0;
+  }
+  
+  .header-list {
+    margin-bottom: 0;
+    display: flex;
+    align-items: stretch; 
+    justify-content: center;
+    width: 100%;
+  }
+  
+  .header-list-item {
+    padding-bottom: 21px;
+    margin: 0 20px;
+    display: block;
+    flex: 0 1 auto; 
+    list-style-type: none;
+    font-size: 20px;
+  }
+  
+  .a-list-item {
+    color: $tier-2-text;
+  }
+  
+  .header-list-item:hover {
+    color: $navbar-inverse-bg;
+    border-bottom: 1.5px solid $navbar-inverse-bg;
+  }
+  
+  .dropdown-toggle {
+    height: 60px;
+  }
+  
+  .dropdown-toggle:active, .open .dropdown-toggle {
+    background: none !important; 
+    color: $brand-success !important;
+  }
+  
+  .navbar-default .navbar-toggle {
+      border-color: $navbar-inverse-link-color ! important;
+  }
+  
+  .navbar-default .navbar-toggle .icon-bar {
+      background-color: $navbar-inverse-link-color ! important;
```

Comment:
Be consistent with spaces, either 2 or 4 spaces.

Reply:
2 spaces would be best and consistent with the rest of the widgets & linting rules. See the EditorConfig. https://github.com/platform-experience/serviceportal-widget-library/blob/master/.editorconfig